pkg/ctl/enable/profile.go

```diff
@@ -27,13 +27,14 @@ import (
 
 type options struct {
 	gitOptions           git.Options
-	quickstartNameArg    string
+	profileName          string
```

Comment:
This is not really the name of the profile. It can be its URL. How about `profileNameArg`?